examples/production/values.yaml

```diff
@@ -19,6 +19,8 @@ spire-server:
     allowPrivilegeEscalation: false
     runAsNonRoot: true
     readOnlyRootFilesystem: true
+    runAsUser: 1000
+    runAsGroup: 1000
```

Comment:
I was able to run the tests without these two changes. So we can get rid of them.